pages/linux/slapt-src.md

```diff
@@ -0,0 +1,24 @@
+# slapt-src
+
+> A utility to automate building of slackbuilds.
+> SlackBuild sources need to be configured in the slapt-srcrc file.
```

Comment:
Typo here? `slapt-srcrc`?

Reply:
No, not a typo. That's the name of the configuration file: `slapt-srcrc`